### DISCUSSION TITLE
Backup / Borg

### DISCUSSION THREAD

**Discussion by manfahrer** [nextcloud/all-in-one#1546] [Questions] [answered]
Hello can you tell me how many days the backup backs up in the past. Will old backups be deleted or I have to do it myself?

**Comment by szaimen** [comment, marked as answer]
Hi, they get pruned regularly with these options: https://github.com/nextcloud/all-in-one/blob/756511f16afaac36f0d78d80891ec81c656878fa/Containers/borgbackup/backupscript.sh#L150 See https://borgbackup.readthedocs.io/en/stable/usage/prune.html for an explanation.

So there is no need to do anything manually.
